public/directives/wz-logtest/components/logtest.tsx

```diff
@@ -91,6 +103,7 @@ export const Logtest = compose(
             log_format: 'syslog',
             location: 'logtest',
```

Comment:
Maybe it is not related to the issue, but all the logs have these properties. The user can't try the Ruleset test utility with another type of log. If we give that possibility to select the log type, we should refactor the input log, replacing the text area for a form, with the log and type (i am not sure if we should include location too).

More info about another type of logs: https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#log-format